validations/scripts/utils/config.py

```diff
@@ -0,0 +1,293 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+from enum import Enum
+from pathlib import Path
+from typing import Union, Dict, Optional
+
+CONFIG_MODULE_DIR = Path(__file__).resolve().parent
+
```

Comment:
This file has a lot of data specific to Provisioner.
It should not be part of `validations/scripts/utils/config.py`
It should be maintained in Provisioner API.
`validations` directory should be independent of Provisioner and Salt. It is supposed to be stand alone, no hard bindings